config/jobs/kubernetes-incubator/service-catalog/service-catalog-presubmits.yaml

```diff
@@ -18,3 +18,37 @@ presubmits:
         env:
         - name: NO_DOCKER
           value: "1"
+  - name: pull-service-catalog-xbuild
+    agent: kubernetes
+    context: pull-service-catalog-xbuild
+    always_run: true
+    skip_report: false
+    rerun_command: "/test pull-service-catalog-xbuild"
+    trigger: "(?m)^/test (all|pull-service-catalog-xbuild)\\s*"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180716-9145034c9-1.10
+        args:
+        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
```

Comment:
this should be github.com/kubernetes-incubator instead of sigs.k8s.io

Reply:
of course it should 🙃

Reply:
followup fix in https://github.com/kubernetes/test-infra/pull/8763